index.html

```diff
@@ -0,0 +1,291 @@
+                <div class="content">
+                    <h2 class="topic">
+                        Multi-Post
+                        <br />
+                        Stories
+                    </h2>
+                    <div class="sub-topic">
+                        <span>CANOPY</span>
+                        <span class="dot"></span>
+
+                        <span>Back End Dev</span>
+                        <span class="dot"></span>
+
+                        <span>2015</span>
+                    </div>
+                    <p class="description">
+                        A daily selection of privately personalized reads; no accounts
+                        or sign-ups required.
+                    </p>
+                    <div class="button-links">
+                        <ul>
+                            <li>html</li>
+                            <li>css</li>
+                            <li>javascript</li>
+                        </ul>
+                    </div>
+                    <a href="">
+                        <div class="project">See Project</div>
+                    </a>
+                </div>
+                <img class="img" src="image/card-4.svg" alt="" />
+            </div>
+        </div>
+    </section>
+    <section class="aboutMee">
+        <div id="about" class="myself">
+            <div class="left">
+                <p class="description">
+                <p class="topic">About <br /> Myself</p>
+                <p>
+                Hello I'm a software developer! I can help you build a product ,
+                feature or website Look through some of my work and experience! If
+                you like what you see and have a project you need coded, don't
+                hestiate to contact me.
+                </p>
+                <div class="menuSection">
+                    <a href="#" target="_blank">LET'S CONNECT</a>
+                    <div class="socialMedia">
+                        <ul class="socialMedia">
+                            <li>
+                                <a href="">
+                                    <img src="image/Twitter.png" alt="" />
+                                </a>
+                            </li>
+                            <li>
+                                <a href="">
+                                    <img src="image/Linkedin.png" alt="" />
+                                </a>
+                            </li>
+                            <li>
+                                <a href="">
+                                    <img src="image/Meduim.png" alt="" />
+                                </a>
+                            </li>
+                            <li>
+                                <a href="">
+                                    <img src="image/Github.png" alt="" />
+                                </a>
+                            </li>
+                            <li>
+                                <a href="">
```

Comment:
- [ ] Good job adding the background color to the section 👍🏼  but I can see some extra margins by the side of the container while the contents of the section are not properly aligned as shown in the Figma design. Kindly fix that 😄 

| Figma Design  | Your Design |
| ------------- | ------------- |
|![image](https://user-images.githubusercontent.com/63473584/185765280-5d58e1d2-3f56-4e3d-a327-7997e19908df.png) |![image](https://user-images.githubusercontent.com/63473584/185765325-d3f3bcba-1ffe-4cf4-969a-4cf1da19d308.png)  |

Reply:
As you may know, I am running out of time and I see you've added some reviews that weren't included in the previous review which isn't the normal procedure